docs/pages/ViewManager/VM-View-Range.md

```diff
@@ -16,24 +16,50 @@ Batch create Drawing Sheets, place Views based on a pre-defined template, and ea
 
 ---
 
-# View Templates tab
+# View Range tab
 
-Manage all View Templates in this tab.
+Manage the View Range of your views individually or in batch in one place.
```

Comment:
"Manage the View Range of your Views individually or in batches in one place."

Reply:
Fixed